src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/ChainFlatteningRewriter.cs

```diff
@@ -19,65 +17,12 @@ namespace Microsoft.Health.Fhir.SqlServer.Features.Search.Expressions.Visitors
     /// </summary>
     internal class ChainFlatteningRewriter : SqlExpressionRewriterWithInitialContext<(TableExpression containingTableExpression, int chainLevel)>
```

Comment:
>(TableExpression containingTableExpression, int chainLevel) [](start = 85, length = 59)

Probably not related to PR, but I'm scratching my head when I look on this type.

I've just change it to object and one test which I ran which hits this part is passing. 
What's the point of this complex type if it doesn't used anywhere?

Reply:
Good catch (I assume you mean the `<(TableExpression containingTableExpression, int chainLevel)>` type parameter). It was used before, but not anymore. It can be reverted to object.

Reply:
Yep, sorry for confusion, I was under impression I selected part of line with type, not whole line, but I guess I fat fingered it. (or fat moused if that is the thing)

---
In reply to: [553619485](https://github.com/microsoft/fhir-server/pull/1569#discussion_r553619485) [](ancestors = 553619485)

Reply:
Fixed in https://github.com/microsoft/fhir-server/pull/1569/commits/427a33aba193c6f5c6c9fe659774fc90da51e64f